test/e2e/capabilityargocd_test.go

```diff
@@ -110,6 +112,62 @@ func assertArgoCDCreated(ctx context.Context, t *testing.T, cfg *envconf.Config)
 	return ctx
 }
 
+func assertArgoCDUpdated(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+	updatedRevision := "updatedRevision"
+	paas := getPaas(ctx, paasWithArgo, t, cfg)
+	paas.Spec.Capabilities = api.PaasCapabilities{
+		"argocd": api.PaasCapability{
+			Enabled:          true,
+			SshSecrets:       map[string]string{paasArgoGitUrl: paasArgoSecret},
+			GitUrl:           paasArgoGitUrl,
+			GitPath:          paasArgoGitPath,
+			GitRevision:      updatedRevision,
+			ExtraPermissions: true,
+		},
+	}
+
+	if err := updatePaasSync(ctx, cfg, paas); err != nil {
+		t.Fatal(err)
+	}
+
+	// As no reconciliation of a PaasNs is triggered, we sleep for safety
+	time.Sleep(10 * time.Second)
+	argoAppSet := getOrFail(ctx, "argoas", "asns", &argo.ApplicationSet{}, t, cfg)
+	entries, _ := getApplicationSetListEntries(argoAppSet)
+
+	// For now this still applies, later we move the git_.. properties to the appSet as well
+	// Assert AppSet entry unchanged
+	assert.Len(t, entries, 1, "ApplicationSet contains one List generator")
+	assert.Equal(t, map[string]string{
+		"paas":       paasWithArgo,
+		"requestor":  "paas-requestor",
+		"service":    "paas",
+		"subservice": "capability",
+	}, entries[0], "ApplicationSet List generator contains the correct parameters")
+
+	assert.NotNil(t, getOrFail(ctx, paasArgoNs, corev1.NamespaceAll, &corev1.Namespace{}, t, cfg), "ArgoCD namespace created")
+
+	// Assert ArgoCD unchanged
+	argocd := getOrFail(ctx, "argocd", paasArgoNs, &v1beta1.ArgoCD{}, t, cfg)
+	assert.Equal(t, paas.UID, argocd.OwnerReferences[0].UID)
+	assert.Equal(t, "role:tester", *argocd.Spec.RBAC.DefaultPolicy)
+	assert.Equal(t, "g, system:cluster-admins, role:admin", *argocd.Spec.RBAC.Policy)
+	assert.Equal(t, "[groups]", *argocd.Spec.RBAC.Scopes)
+
+	// Assert Bootstrap is not updated as described in issue #185
+	applications := listOrFail(ctx, paasArgoNs, &argo.ApplicationList{}, t, cfg).Items
+	assert.Len(t, applications, 1, "An application is present in the ArgoCD namespace")
+	assert.Equal(t, "paas-bootstrap", applications[0].Name)
+	assert.Equal(t, argo.ApplicationSource{
+		RepoURL:        paasArgoGitUrl,
+		Path:           paasArgoGitPath,
+		TargetRevision: paasArgoGitRevision,
```

Comment:
Once the bug is fixed, this will be changed to `updatedRevision`?

Reply:
It must be after fixing, maybe better to do an assert.NotEqual now?